### ISSUE TITLE
111const-deflate.t running more tests than expected

### ISSUE THREAD

**Issue by amurf** [pmqs/Archive-Zip-SimpleZip#13]
It looks like after this change upstream https://github.com/pmqs/IO-Compress/commit/6e877cce4757d368a8ccffbb0fad84dca9e05a26

`111const-deflate.t` test is now failing, as more constants now exist, so an extra 4 tests are running and throwing out the amount of expected tests to run.

Output when attempting to install

```
# perl                      5.030003
# Archive::Zip::SimpleZip   0.035
# Archive::Zip::SimpleUnzip 0.035
# IO::Compress::Base        2.096
# IO::Compress::Zip         2.096
# IO::Compress::Lzma        Not Installed
# IO::Uncompress::Base      2.096
# IO::Uncompress::Unzip     2.096
# IO::Uncompress::UnLzma    Not Installed
# Compress::Raw::Zlib       2.096
# Compress::Raw::Bzip2      2.096
# Compress::Raw::Lzma       Not Installed
# zlib                      1.2.11
# bzip2                     1.0.8, 13-Jul-2019

# Looks like you planned 83 tests but ran 4 extra
```

**Comment by pmqs** [comment]
Thanks @amurf , I'll get that sorted

**Comment by arc** [comment]
~Just FYI, this seems to be caused by a newer version of Compress::Raw::Zlib exporting more constants than the ::SimpleZip test suite expects.~

~Apologies for the noise if you'd already come to the same conclusion.~

Uh, you had — sorry, I didn't spot your commit :-(

**Comment by pmqs** [comment]
Should be fixed in version 0.036 . Update uploaded to CPAN
